github/lib/configure_repos.rb

```diff
@@ -34,14 +35,28 @@ def remove_old_webhooks!
       end
   end
 
+  def verify_repo_tags!
```

Comment:
I'd suggest renaming this to:

```suggestion
  def verify_repo_tags
```

Use of method names ending in `!` are usually 'write' operations (the examples elsewhere in this file are all 'doing things' like removing webhooks). Your method is a read-only one, so omitting the `!` is appropriate.

As an aside, conventions of using or not using `!` are not very consistently enforced outside of major libraries like ActiveSupport (which does have quite well understood variants of methods like [`save` vs `save!`](https://stackoverflow.com/questions/5056385/when-to-use-save-vs-save-in-model), one of which will fail 'silently', for example). It's cool we can make methods ending in `!` but it's not something we actually do a lot of on GOV.UK 🤔